src/components/base/JInputField/JInputField.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+
+import React from 'react'
+import classNames from 'classnames'
+
+import { getErrorMessage } from 'utils/form'
+import { JFieldMessage } from 'components/base'
+
+import jInputFieldStyle from './jInputField.m.scss'
+
+export type JInputType = 'text' | 'password'
+export type JInputTheme = 'white'
+
+type Props = StyledComponent<JInputTheme> & {
+  disabled: boolean,
+  infoMessage: string,
+  input: FinalFormInput,
+  label: string,
+  meta: FinalFormMeta,
+  placeholder: string,
+  type: JInputType,
+  validateType: FinalFormValidateType,
+}
+
+function JInputField({
+  className,
+  theme,
+  disabled,
+  infoMessage,
+  input,
+  label,
+  meta,
+  placeholder,
+  type,
+  validateType,
+  ...rest
+}: Props) {
+  const textInput = React.createRef()
+
+  const errorMessage = getErrorMessage(meta, validateType)
+  const hasLabel = !!label
+  const hasPlaceholder = !!placeholder
+  const hasError = !!errorMessage
+  const hasInfo = !!infoMessage
+  const hasValue = !!input.value
+  const isActive = meta.active || hasValue || (hasLabel && hasPlaceholder)
+
+  return (
+    <div
+      className={classNames(
+        jInputFieldStyle.core,
```

Comment:
Now we are adding the plain-text classes for functional tests:
Form of this classes: `__%some-element-name%`
https://github.com/jibrelnetwork/jwallet-web/pull/707/files#diff-8a1bb7f77977752c184f4c2b841e6a2fR104